docs/tp_templates/pages/root_page.md

```diff
@@ -0,0 +1,128 @@
+!!! note "Available in Taipy Enterprise edition"
+
+    This section is relevant only to the [Taipy Enterprise Edition](https://taipy.io/enterprise).
+
+    [Contact us](https://taipy.io/book-a-call){: .tp-btn .tp-btn--accent target='blank' }
+
+In a Taipy application, the root page defines the overall layout of the application. During the
+development process of a Taipy application, adding a root page is a common practice to provide an
+entry point for the application.
+
+To reduce the development time, Taipy provides a root page template, which is designed as a best
+practice page for a Taipy root page.
+
+Out-of-the-box, the root page defines the overall layout of the application, including the title,
+navigation bar, and the content area.
+
+This page template offers several key benefits:
+
+- **Ease of Use**: The page template is designed to be user-friendly, with a simple CLI
+    interface that guides developers through the page creation process on top of existing
+    application. The pages then can be easily customized and plugged into the application.
+- **Customization**: The template provides high flexibility and customization options on page
+    creation, allowing developers to tailor the page to meet specific requirements and use cases.
+
+# How to create a page
+
+<!-- TODO: The root page should be the final page of the application. -->
+
+To create the page from the root page template, change to the folder of the application in which
+you want to create the page and run `taipy create --page root` from the CLI. Then answer a few
+questions to customize your page.
+
+```console
+$ taipy create --page root
+  [1/4] Page title (root):
+  [2/4] The folder that contains the pages (pages):
+  [3/4] Application title (Taipy Application):
+  [4/4] Variable that contains the dictionary of application pages (pages):
```

Comment:
- Don't expose implementation details (dict - nobody cares)
- "application pages"  -> "pages" (what else could this be?)